Makefile

```
@@ -183,7 +183,9 @@ status-go-deps:
 statusgo-android: generate
 statusgo-android: ##@cross-compile Build status-go for Android
 	@echo "Building status-go for Android..."
-	export GO111MODULE=off; \
+	mkdir -p build/bin \
+	export GO111MODULE=on; \
+		export GOFLAGS="-mod=mod" \
```

What's the point of this indent? Also you don't need to exports:
```suggestion
	export GO111MODULE=on GOFLAGS="-mod=mod"; \
```

---

ah sorry, i fucked up indents trying to get this PR out quickly.
will fix.